proto/did/did.proto

```diff
@@ -30,12 +30,12 @@ message DidDocument {
   // such as cryptographic public keys, which can be used 
   // to authenticate or authorize interactions with the DID subject or associated parties.
   // https://www.w3.org/TR/did-core/#verification-methods
-  repeated VerificationMethod verificationMethods  = 4;
+  repeated VerificationMethod verificationMethod  = 4;
 
   // Services are used in DID documents to express ways of communicating 
   // with the DID subject or associated entities.
   // https://www.w3.org/TR/did-core/#services
-  repeated Service services                        = 5;
+  repeated Service service                        = 5;
```

Comment:
I am not sure about this change, because in the generated it would look like:

[]Service service

for example:

for _, service := range service

but I would like it to be in plural. What do you think?

Reply:
I agree with you, I like the plural more, but the [specs have it singular](https://www.w3.org/TR/did-core/#services) an it will require to add add customizations like:

```
repeated Service services = 5 [json_name="service", (gogoproto.jsontag) = "service,omitempty"]
```
that I prefer not to include